test/BuyTshirt.spec.ts

```diff
@@ -1,49 +1,68 @@
+      const menuContentPage: MenuContentPage = new MenuContentPage();
+      const productDetailPage: ProductDetailPage = new ProductDetailPage();
+      const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
+      const summaryStepPage: SummaryStepPage = new SummaryStepPage();
+      beforeAll(async () => {
 
-    await menuContentPage.goToTShirtMenu();
+        await menuContentPage.goToTShirtMenu();
+        await productDetailPage.goToProductDetail();
+        await productAddedModalPage.goToProductAddedModal();
 
-    await productDetailPage.goToProductDetail();
+        const expectedCondition = protractor.ExpectedConditions;
+        const condition = expectedCondition
+            .elementToBeClickable(summaryStepPage.getSummaryStepPage);
+        await browser.wait(condition, 3000, 'error waiting');
+        await summaryStepPage.goToSummaryStep();
 
-    await productAddedModalPage.goToProductAddedModal();
+      });
 
-    var expectedCondition = protractor.ExpectedConditions;
-    var condition= expectedCondition.elementToBeClickable(summaryStepPage.getSummaryStepPage);
-    await browser.wait(condition, 3000, 'error waiting');
-    await summaryStepPage.goToSummaryStep();
+      describe('Logeo en la aplicación', () => {
+        const signInStepPage: SignInStepPage = new SignInStepPage();
+        const addressStepPage: AddressStepPage = new AddressStepPage();
+        beforeAll(async () => {
 
-    await signInStepPage.goToSignInStep();
+          await signInStepPage.goToSignInStep();
+          await signInStepPage.addDataSignIn();
+          await addressStepPage.goToAddressStep();
 
-    await signInStepPage.addDataSignIn();
-    await addressStepPage.goToAddressStep();
+        });
 
-    await shippingStepPage.goToShippingStep();
+        describe('Seleccionar la dirección por defecto', () => {
+          const shippingStepPage: ShippingStepPage = new ShippingStepPage();
+          beforeAll(async () => {
 
-    await shippingStepPage.checkBoxShipping();
+            await shippingStepPage.goToShippingStep();
+            await shippingStepPage.checkBoxShipping();
 
-    await paymentStepPage.goToPaymentStep();
+          });
 
-    await bankPaymentPage.goToBankPayment();
+          describe('Pago en el banco', () => {
+            const paymentStepPage: PaymentStepPage = new PaymentStepPage();
+            const bankPaymentPage: BankPaymentPage = new BankPaymentPage();
+            const orderResumePage: OrderResumePage = new OrderResumePage();
+            it('then should be bought a t-shirt', async () => {
 
-    await orderResumePage.goToOrderResume();
-
-    await orderResumePage.checkText();
+              await paymentStepPage.goToPaymentStep();
+              await bankPaymentPage.goToBankPayment();
+              await orderResumePage.goToOrderResume();
+              await expect(orderResumePage.inicialText.getText())
+            .toBe('Your order on My Store is complete.');
```

Comment:
Muy bien hecho, lo único que te falta es dejar el "it" unicamente para la prueba, es decir, el it se hace unicamente para la comprobación del resultado (expect).